sql/core/benchmarks/FilterPushdownBenchmark-jdk21-results.txt

```diff
@@ -2,733 +2,733 @@
 Pushdown for many distinct value case
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
+OpenJDK 64-Bit Server VM 21.0.4+7-LTS on Linux 6.5.0-1025-azure
 AMD EPYC 7763 64-Core Processor
 Select 0 string row (value IS NULL):      Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Parquet Vectorized                                 6309           6370          57          2.5         401.1       1.0X
-Parquet Vectorized (Pushdown)                       294            324          23         53.4          18.7      21.4X
-Native ORC Vectorized                              5129           5216          60          3.1         326.1       1.2X
-Native ORC Vectorized (Pushdown)                    323            330           6         48.7          20.5      19.5X
+Parquet Vectorized                                 6345           6437          61          2.5         403.4       1.0X
+Parquet Vectorized (Pushdown)                       341            363          12         46.2          21.7      18.6X
```

Comment:
`Parquet Vectorized (Pushdown)` becomes slower consistently in this benchmark for both Java 17/21.